src/App/App.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { BrowserRouter, Routes, Route, Navigate } from 'react-router-dom';
+
+import { RepoPage } from './pages/RepoPage';
+import { Repositories } from './pages/Repositories';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Repositories />} />
+        <Route path="/repo/:owner" element={<Repositories />} />
```

Comment:
введенное значение лучше реализовать через query-параметр, то есть /?search={owner}, и тогда тут будет одна страница
```suggestion
        <Route path="/" element={<Repositories />} />
```